roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -20,7 +21,7 @@ def is_active(cls, task_vars):
     @staticmethod
     def is_first_master(task_vars):
```

Comment:
This is a bad name, makes the `is_active` expression look like we only care about being first master, when in fact it is a bit more than that.